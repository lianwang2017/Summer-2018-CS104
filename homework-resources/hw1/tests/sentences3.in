great intelligent
minds men women people
think look sound
alike cool