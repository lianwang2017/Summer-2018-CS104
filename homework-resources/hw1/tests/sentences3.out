The minds think.
The minds think alike.
The minds think cool.
The minds look.
The minds look alike.
The minds look cool.
The minds sound.
The minds sound alike.
The minds sound cool.
The men think.
The men think alike.
The men think cool.
The men look.
The men look alike.
The men look cool.
The men sound.
The men sound alike.
The men sound cool.
The women think.
The women think alike.
The women think cool.
The women look.
The women look alike.
The women look cool.
The women sound.
The women sound alike.
The women sound cool.
The people think.
The people think alike.
The people think cool.
The people look.
The people look alike.
The people look cool.
The people sound.
The people sound alike.
The people sound cool.
The great minds think.
The great minds think alike.
The great minds think cool.
The great minds look.
The great minds look alike.
The great minds look cool.
The great minds sound.
The great minds sound alike.
The great minds sound cool.
The great men think.
The great men think alike.
The great men think cool.
The great men look.
The great men look alike.
The great men look cool.
The great men sound.
The great men sound alike.
The great men sound cool.
The great women think.
The great women think alike.
The great women think cool.
The great women look.
The great women look alike.
The great women look cool.
The great women sound.
The great women sound alike.
The great women sound cool.
The great people think.
The great people think alike.
The great people think cool.
The great people look.
The great people look alike.
The great people look cool.
The great people sound.
The great people sound alike.
The great people sound cool.
The intelligent minds think.
The intelligent minds think alike.
The intelligent minds think cool.
The intelligent minds look.
The intelligent minds look alike.
The intelligent minds look cool.
The intelligent minds sound.
The intelligent minds sound alike.
The intelligent minds sound cool.
The intelligent men think.
The intelligent men think alike.
The intelligent men think cool.
The intelligent men look.
The intelligent men look alike.
The intelligent men look cool.
The intelligent men sound.
The intelligent men sound alike.
The intelligent men sound cool.
The intelligent women think.
The intelligent women think alike.
The intelligent women think cool.
The intelligent women look.
The intelligent women look alike.
The intelligent women look cool.
The intelligent women sound.
The intelligent women sound alike.
The intelligent women sound cool.
The intelligent people think.
The intelligent people think alike.
The intelligent people think cool.
The intelligent people look.
The intelligent people look alike.
The intelligent people look cool.
The intelligent people sound.
The intelligent people sound alike.
The intelligent people sound cool.
108 sentences.
